shipyard_blueprints/sqlserver/shipyard_sqlserver/errors/exceptions.py

```diff
@@ -1,9 +1,5 @@
 from shipyard_templates import ExitCodeException, Database
```

Comment:
I don't think there's a need for errors to be in a separate folder with on file. Should move it up a level to match the other blueprints